modules/understanding-upgrade-channels.adoc

```diff
@@ -7,79 +7,89 @@
 // * updating/updating-disconnected-cluster.adoc
 
 [id="understanding-upgrade-channels_{context}"]
-= Understanding {product-title} upgrade channels
+= {product-title} Upgrade Channels and Update Types
 
-In {product-title} 4.1, Red Hat introduced the concept of upgrade channels for
-recommending the appropriate upgrade versions to your cluster. Upgrade channels
-separate upgrade strategies and also are used to control the cadence of updates.
-Channels are tied to a minor version of {product-title}. For instance,
-{product-title} {product-version} channels will never include an upgrade to a
+In {product-title} 4.1, Red Hat introduced the concept of Upgrade Channels for
+recommending the appropriate upgrade versions to your cluster. Upgrade Channels
+separate upgrade strategies and are also used to control the cadence of updates.
+Upgrade Channels are tied to a minor version of {product-title}. For instance,
+{product-title} {product-version} Upgrade Channels will never include an upgrade to a
 4.5 release. This ensures administrators make an explicit decision to upgrade to
-the next minor version of {product-title}. Channels only control updates and
+the next minor version of {product-title}. Upgrade Channels only control updates and
 have no impact on the version of the cluster you install; the
 `openshift-install` binary for a given patch level of {product-title} always
 installs that patch level.
 
-{product-title} {product-version}, which includes the upgrade from the previous 4.3
-release, has three upgrade channels to choose from:
+The updates within an Upgrade Channel are designated as one of two types. These
```

Comment:
I'd rather call the nodes "releases" and not "updates".  If we only ever cut a single release (with no update edges), it would still be a release, but it wouldn't update anything.